woocommerce/automations
# Attach changelog to release automatically

This PR fixes woocommerce/woocommerce-gutenberg-products-block#4636

## Manual test

1. Create a new repo. You can start from [this template](https://github.com/gigitux/woo-test-repo).
2. Set a workflow for `woocoomerce-automation`. You can copy-paste [this file](https://github.com/gigitux/woocommerce-automation-test-repo/blob/main/.github/workflows/automation.yml). The important thing is to change the value `uses`. It has to point to this branch.
3. Create a new milestone and give the title a number version (e.g.: 1.0).
4. Create a PR with a changelog, give the name `Release: 1.0`([example](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6129)). It would be great if you copy the entire PR body of the linked PR.
5. Create a release with Github UI, and give as the release title the version (for example 1.0).
6. Now, check that the action updates the release description and put the changelog.
